xla/service/gpu/gpu_compiler_test.cc

```diff
@@ -1405,8 +1405,6 @@ TEST_F(PassOrderTest, PassesAreRunInCorrectOrder) {
                   /*last_pass_regex=*/"priority-fusion");
   VerifyPassOrder(/*first_pass_regex=*/"layout-assignment",
                   /*last_pass_regex=*/"layout_normalization");
-  VerifyPassOrder(/*first_pass_regex=*/"host-offload-legalize",
```

Comment:
Unfotunately, the pass order test cannot quite express that we want to run host-offload-legalize just before each layout normalization, so I had to relax the test.

Reply:
Why not? You could write a custom test that checks that each time `layout-normalization` is run, it is preceded by `host-offload-legalize`?

Reply:
Done.